integration-cli/docker_cli_build_test.go

```diff
@@ -3772,6 +3722,7 @@ func (s *DockerSuite) TestBuildFromMixedcaseDockerfile(c *check.C) {
 	})
 }
 
+// FIXME(vdemeester) should migrate to docker/cli tests
```

Comment:
We should verify that we have test-coverage for the server-side "building from URL" (which is also still a thing), and all incantations of how we send the build-context (dockerfile within context/outside of build context etc)

Reply:
definitely 😉